protocol/streaming/full_node_streaming_manager.go

```diff
@@ -134,6 +139,17 @@ func (sm *FullNodeStreamingManagerImpl) Subscribe(
 		messageSender:  messageSender,
 		updatesChannel: make(chan []clobtypes.StreamUpdate, sm.maxSubscriptionChannelSize),
 	}
+	for _, clobPairId := range clobPairIds {
+		// if clobPairId exists in the map, append the subscription id to the slice
+		// otherwise, create a new slice with the subscription id
+		if _, ok := sm.clobPairIdToSubscriptionIdMapping[clobPairId]; !ok {
+			sm.clobPairIdToSubscriptionIdMapping[clobPairId] = []uint32{}
+		}
+		sm.clobPairIdToSubscriptionIdMapping[clobPairId] = append(
+			sm.clobPairIdToSubscriptionIdMapping[clobPairId],
+			sm.nextSubscriptionId,
```

Comment:
do we evict stuff from this map when we prune order subscriptions from the fns manager?

Reply:
done